src/utilities/extendSchema.ts

```diff
@@ -393,24 +393,24 @@ export function extendSchemaImpl(
   function getOperationTypes(
     nodes: ReadonlyArray<SchemaDefinitionNode | SchemaExtensionNode>,
   ): {
-    query: Maybe<GraphQLObjectType>;
-    mutation: Maybe<GraphQLObjectType>;
-    subscription: Maybe<GraphQLObjectType>;
+    query?: Maybe<GraphQLObjectType>;
+    mutation?: Maybe<GraphQLObjectType>;
+    subscription?: Maybe<GraphQLObjectType>;
   } {
     const opTypes = {};
     for (const node of nodes) {
       // istanbul ignore next (See: 'https://github.com/graphql/graphql-js/issues/2203')
       const operationTypesNodes = node.operationTypes ?? [];
 
       for (const operationType of operationTypesNodes) {
+        // Note: While this could make early assertions to get the correctly
+        // typed values below, that would throw immediately while type system
+        // validation with validateSchema() will produce more actionable results.
+        // TODO ts-expect-error
```

Comment:
why are there `TODO` for `ts-expect-error`?

Reply:
Because we're not actually turning strict mode on yet so there isn't an error to expect, so TSC fails on seeing an "expect error" without an error. As soon as we turn on strict mode we expect an error here. 

This makes it easy to search and update all of these when we toggle strict on!

Reply:
makes sense thanks!